### PR TITLE
Code of Conduct

### PR DESCRIPTION
**Pull request by nelsonjhk**
Our Code of Conduct, now in markdown. Comments are welcome. An intro paragraph might be added, and once the document's up we should also reference this document in our contributor's guide.

### CODE REVIEW

**Comment by artasparks** [comment]
LGTM. Once this gets in, I'd like to have everyone sign off on the PR. Might want a CLA-ish thing first. We can talk about it in the meeting.

**Review by artasparks — APPROVED** [review]

**Review by jared-nishikawa — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at conduct@otrego.com. All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project’s leadership.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org/), version 1.4, available at [https://www.contributor-covenant.org/version/1/4/code-of-conduct.html](https://www.contributor-covenant.org/version/1/4/code-of-conduct.html), and from the Go Community Code of Conduct, available at [https://golang.org/conduct](https://golang.org/conduct).
+
+## Summary
```
Comment (artasparks) [inline CODE_OF_CONDUCT.md:58]: Dummy comment

**Review by RodDeacon — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+
+# Otrego Code of Conduct
+
+## Our Values
+
+Clear communication is foundational for healthy collaboration. We value the time needed to be thoughtful in how we communicate and value the time contributors offer to the project. We make efforts to counter prejudice and biases against contributors and contributions. Ultimately, we put personal development ahead of production.
+
+As representatives of the go community, we strive for a welcoming environment. We aim to promote growth of the game and expand public knowledge of go, from the board to go’s intersections with technology and self-knowledge.
```
Comment (TamaBrian) [inline CODE_OF_CONDUCT.md]: **Peer Review**:  
     Willingness to volunteer time to test colleagues code.
     Willingness to openly show colleagues ones own code.  
**Collaboration**:  
     A culture of collaboration is paramount to the success of the whole team.  A few minutes of timely assistance may to a stymied colleague may be just what is needed to bring the whole team's project to success.
     Team members come from different backgrounds and can contribute in various complimentary ways.  Still, individuals developing new talents, or honing old talents, may require a helping hand.  For the team's benefit, volunteering some help will benefit the the whole team.
**Mentoring**:

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
```
Comment (TamaBrian) [inline CODE_OF_CONDUCT.md:14]: Whole heartily agree!

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+
+
+## Scope
+
+This Code of Conduct applies within all project spaces, and it also applies when an individual is representing the project or its community in public spaces. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+The English-speaking go community is small, and as contributors and maintainers of a project aimed at go players, remember that we act as ambassadors for the go community as a whole.
```
Comment (TamaBrian) [inline CODE_OF_CONDUCT.md:46]: ... for the **international** go community as a whole.

Comment (artasparks) [reply CODE_OF_CONDUCT.md:46]: I'm not sure that the international go community is in scope -- maybe let's chat in our weekly sync. To me, serving the international go community implies dedicate efforts towards internationalization and localization. I could see that being part of our scope in the future, but that seems challenging or even perhaps hubristic to lead with.

**Review by mhoak — APPROVED** [review]
This looks good. Thanks!

**Review by KiloLimaFoxtrot — APPROVED** [review]
This looks like a good work in progress, and it will grow as we grow as a group.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (nelsonjhk) [reply CODE_OF_CONDUCT.md]:
```suggestion
Clear communication is foundational for healthy collaboration. We value the time needed to be thoughtful in how we communicate and value the time contributors offer to the project. We make efforts to counter prejudice and biases against contributors and contributions, and prioritize transparency in code and code reviews. Ultimately, we put personal development ahead of production.

A culture of collaboration means valuing our contributors' different backgrounds and distinct contributions. We encourage reaching out to offer assistance and taking opportunities for mentorship that arise.

As representatives of the go community, we strive for a welcoming environment. We aim to promote growth of the game and expand public knowledge of go, from the board to go’s intersections with technology and self-knowledge.
```

Thanks, Brian. I added a paragraph on mentorship and collaboration, and I think your point about transparency is important and added a line to the first paragraph to emphasize that as well. What do you think?

**Comment by TamaBrian** [comment]
Jeremy,  Right on.  I fully concur.

     Brian

**Pull request merged**